src/main/java/com/kodilla/ecommercee/domain/Cart.java

```diff
@@ -1,68 +1,39 @@
 package com.kodilla.ecommercee.domain;
 
 import lombok.AllArgsConstructor;
+import lombok.Data;
 import lombok.NoArgsConstructor;
 
 import javax.persistence.*;
 import javax.validation.constraints.NotNull;
 import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
 
 @Entity(name = "carts")
-@Table
-@AllArgsConstructor
 @NoArgsConstructor
+@AllArgsConstructor
+@Data
 public class Cart {
-    private int id;
-    private String name;
-    private String description;
-    private BigDecimal price;
-    private int groupId;
-
     @Id
-    @GeneratedValue
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     @NotNull
-    @Column(name = "Id", unique = true)
-    public int getId() {
-        return id;
-    }
-
-    public void setId(int id) {
-        this.id = id;
-    }
+    @Column(name = "id", unique = true)
+    private Long id;
 
     @Column(name = "name")
-    public String getName() {
-        return name;
-    }
-
-    public void setName(String name) {
-        this.name = name;
-    }
+    private String name;
 
     @Column(name = "description")
-    public String getDescription() {
-        return description;
-    }
-
-    public void setDescription(String description) {
-        this.description = description;
-    }
+    private String description;
```

Comment:
Tak samo, jak wyżej, opis koszyka? Może chodziło o opis produktów, które są w koszyku? Aczkolwiek pytanie, czy opis produktów w koszyku w ogóle nas interesuje?

Reply:
Też sądzę, że chodzi o nazwę produktów w koszyku. Przy nadawaniu nazw pól sugerowałem się plikiem ze specyfikacją projektu cart.spec.json

Reply:
Patrzyłem na ten plik i coś tam nie tak jest, bo w ogóle mi to nie pasuje do tej encji...